eng/pipelines/templates/stages/archetype-sdk-tool-python.yml

```diff
@@ -0,0 +1,126 @@
+parameters:
+- name: PythonVersion
+  type: string
+  default: '3.13'
+- name: PackagePath
+  type: string
+  default: ''
+- name: FeedName
+  type: string
+  default: ''
+- name: ArtifactName
+  type: string
+  default: ''
+- name: PackageName
+  type: string
+  default: ''
+- name: TestSteps
+  type: object
+  default: []
+- name: BuildUsingPyProjectToml
+  type: boolean
+  default: false
+
+
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    stages:
+      - stage: 'Build'
+        variables:
+          - template: /eng/pipelines/templates/variables/globals.yml
+          - template: /eng/pipelines/templates/variables/image.yml
+        jobs:
+          - job: 'Build'
+
+            pool:
+              name: $(LINUXPOOL)
+              image: $(LINUXNEXTVMIMAGE)
+              os: linux
+
+            steps:
+              - template: /eng/pipelines/templates/steps/use-python-version.yml
+                parameters:
+                  versionSpec: '${{ parameters.PythonVersion }}'
+
+              - script: |
+                  python --version
+                  python -m pip install virtualenv setuptools wheel packaging build
+                displayName: 'Setup Python Environment'
+
+              - ${{ if eq(parameters.BuildUsingPyProjectToml, true) }}:
```

Comment:
Is there any reason you're conditioning this? For new tooling like this I'd prefer we just use the new stuff only. (python -m build). We are only maintaining both in `azure-sdk-tools` because of some limitations around `sdist` creation when using `python -m build`, and those don't apply in this situation.